src/app/modules/home/pages/bookmarks/bookmarks.component.ts

```diff
@@ -0,0 +1,52 @@
+import { Component, OnInit } from '@angular/core';
+import { Subscription } from 'rxjs';
+import { ApiService } from 'src/app/core/http/api.service';
+import { Post } from 'src/app/core/models/post.model';
+
+@Component({
+  selector: 'app-bookmarks',
+  templateUrl: './bookmarks.component.html',
+  styleUrls: ['./bookmarks.component.css'],
+})
+export class BookmarksComponent implements OnInit {
```

Comment:
implements AfterContentInit
it is best to keep as in implements
and remove OnInit if you don't want to use it in all components